packages/pulumi-aws/src/apps/gateway/GatewayApp.ts

```diff
@@ -0,0 +1,77 @@
+import { defineApp, createGenericApplication, ApplicationConfig } from "@webiny/pulumi-sdk";
+
+import { createReactAppGateway, GatewayReactAppConfig } from "./GatewayReactApp";
+import { createLambdas } from "./GatewayLambdas";
+import { createApiAppGateway, GatewayApiConfig } from "./GatewayApi";
+
+export interface GatewayAppConfig {
+    api?: GatewayApiConfig;
+    admin?: GatewayReactAppConfig;
+    website?: GatewayReactAppConfig;
+}
+
+export const GatewayApp = defineApp({
+    name: "Gateway",
+    config(app, config: GatewayAppConfig) {
+        const lambdas = createLambdas(app);
+
+        const api = createApiAppGateway(app, {
```

Comment:
Once question here... 

I see there are three `createXYZGateway` calls, where internally each deploys a separate CloudFront distribution. But in my mind, if I remember correctly, there was only one gateway CDN deployed, that acts as a gateway.

Is the current code just temporary? Is this just because WCP hasn't yet been completed? 

Additional info would be useful.

Reply:
No, each application has its own domain and its own cloudfront distro that splits the traffic. It basically mimics current setup. We don't have API/admin/website on a single domain and CloudFront, so traffic splitting gateway works the same.

Reply:
Awesome, now it's clear. Thanks! 👍🏻